src/main/java/com/suse/saltstack/netapi/config/ClientConfig.java

```diff
@@ -86,5 +86,4 @@ public Key(T defaultValue) {
        Object value = store.get(key);
        return value != null ? (T)value : key.defaultValue;
```

Comment:
It is mostly my personal preference it seems, I found examples doing it with no blank after the `)` as well, just like above. However, at least at [google] (https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.6.2-horizontal-whitespace) they seem to put blanks there (see 5.).